src/forklift/core.py

```diff
@@ -225,7 +235,7 @@ def _hash(crate, hash_path, needs_reproject):
     if not arcpy.Exists(path.join(hash_path, crate.name)):
         log.debug('%s does not exist. creating', crate.name)
         table = arcpy.CreateTable_management(hash_path, crate.name)
-        arcpy.AddField_management(table, hash_id_field, 'TEXT', field_length=50)
+        arcpy.AddField_management(table, hash_id_field, 'LONG')
```

Comment:
So what about [this line](https://github.com/agrc/forklift/blob/c1dea964933299867f89f0a73265480fb67d4cd0/src/forklift/core.py#L182)? The hash key should always be the destination id, right? Here's the source issue that prompted that line: https://github.com/agrc/forklift/issues/150.